spec/models/contributions_calendar_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe Wpcc::ContributionsCalendar, type: :model do
+  let(:contributions_calendar) do
+    described_class.new(
+      eligible_salary: eligible_salary,
+      employee_percent: employee_percent,
+      employer_percent: employer_percent,
+      salary_frequency: salary_frequency
+    )
+  end
+
+  describe '#schedule' do
+    let(:eligible_salary) { 0 }
+    let(:employee_percent) { 0 }
+    let(:employer_percent) { 0 }
+    let(:salary_frequency) { 0 }
+    let(:period_contribution_calculator) do
+      double('PeriodContributionCalculator')
+    end
+
+    subject(:schedule) { contributions_calendar.schedule }
+
+    it 'returns an Array of PeriodContribution objects' do
+      expect(schedule).to be_an(Array)
+      expect(schedule.size).to_not be_zero
+      schedule.each do |period|
+        expect(period).to be_a Wpcc::PeriodContribution
+      end
+    end
+
+    context 'creates PeriodContribution objects for each period' do
```

Comment:
This is more of a description rather than a context definition. 

```
describe 'creates PeriodContribution objects for each period' do
  context 'when the employee/employer percents ARE in the yml file' do
    ...
  end
end
```